recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "0.3" %}
+{% set version = "0.5" %}
 
 package:
   name: pytest-mpl
   version: {{ version }}
 
 source:
   fn: pytest-mpl-{{version}}.tar.gz
-  url: https://pypi.python.org/packages/source/p/pytest-mpl/pytest-mpl-{{version}}.tar.gz
-  md5: 45912acdd8ae63358711de431ac076ec
+  url: https://pypi.python.org/packages/cb/db/a7b8c2bb859b4f021e349adbebe6968036dd6f236a98be795c1fe621d340/pytest-mpl-{{version}}.tar.gz
```

Comment:
Can you change that ugly new PyPI URL to `https://pypi.io/packages/source/p/pytest-mpl/pytest-mpl-{{ version }}.tar.gz`please?